lib/rubocop/cop/rails/inverse_of.rb

```diff
@@ -65,6 +65,16 @@ module Rails
       #              inverse_of: false)
       #   end
       #
+      # @example IgnoreScopes: true
+      #   # good
+      #   class Blog < ApplicationRecord
+      #     has_many :posts, -> { order(published_at: :desc) }
+      #   end
+      #
+      #   class Post < ApplicationRecord
+      #     belongs_to :blog
+      #   end
+      #
```

Comment:
Perhaps the contrast will be more noticeable whether only the differences are extracted as shown below.

```ruby
      # @example IgnoreScopes: false (default)
      #   # bad
      #   class Blog < ApplicationRecord
      #     has_many :posts, -> { order(published_at: :desc) }
      #   end
      #
      # @example IgnoreScopes: true
      #   # good
      #   class Blog < ApplicationRecord
      #     has_many :posts, -> { order(published_at: :desc) }
      #   end
```

And can you move this as the bottom example?

Reply:
Done! Thank you for the review, and hi 👋🏻!